--- board/freescale/mx28evk/mx28evk.c	2020-07-01 22:26:16.723387000 +0800
+++ board/zlg/imx280a/imx280a.c	2020-07-01 19:36:32.000000000 +0800
@@ -48,12 +48,6 @@
 	gpio_direction_output(MX28_PAD_AUART2_RX__GPIO_3_8, 1);
 #endif
 
-	/* Power on LCD */
-	gpio_direction_output(MX28_PAD_LCD_RESET__GPIO_3_30, 1);
-
-	/* Set contrast to maximum */
-	gpio_direction_output(MX28_PAD_PWM2__GPIO_3_18, 1);
-
 	return 0;
 }
 
